### PR TITLE
Paramiko might not come standard everywhere

### PR DESCRIPTION
**Pull request by abadger**
There is a platform where paramiko isn't shipped but a special version
of paramiko just for our use is shipped.  This code imports paramiko
from that location.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
various networking code

@Qalthos @privateip

### CODE REVIEW

**Review by jamescassell — APPROVED** [review]
shipit

File: lib/ansible/plugins/connection/paramiko_ssh.py [hunk 1]
```diff
@@ -176,7 +176,12 @@
     try:
         import paramiko
         HAVE_PARAMIKO = True
-    except (ImportError, AttributeError) as err:  # paramiko and gssapi are incompatible and raise AttributeError not ImportError
+    except ImportError:
+        try:
+            import ansible_paramiko as paramiko
```
Comment (abadger) [inline lib/ansible/plugins/connection/paramiko_ssh.py:181]:
```suggestion
            import ansible_paramiko as paramiko
            HAVE_PARAMIKO = True
```

**Comment by abadger** [comment]
nitzmahone had a look at this and okayed it earlier on slack.

@Qalthos did you want to have a look?  I need to merge this tomorrow as it blocks the alpha release.

**Review by Qalthos — APPROVED** [review]

**Comment by abadger** [comment]
@Qalthos Thanks!

**Pull request merged**